sig-scalability/configs-and-limits/thresholds.md

```diff
@@ -43,21 +43,23 @@ There are couple caveats to the thresholds we are presenting below:
-| #Pods                  | 3000                      | 150000                   |
-| #Pods per node         | min(110, 10*#cores)       | min(110, 10*#cores)      |
-| #Services              | 5000                      | 10000                    |
-| #All service endpoints | TBD                       | TBD                      |
-| #Endpoints per service | 250                       | n/a                      |
-| #Secrets               | TBD                       | TBD                      |
-| #ConfigMaps            | TBD                       | TBD                      |
-| #Deployments           | 2000                      | TBD                      |
-| #DaemonSets            | TBD                       | TBD                      |
-| #Jobs                  | TBD                       | TBD                      |
-| #StatefulSets          | TBD                       | TBD                      |
+| Quantity                    | Threshold scope=namespace | Threshold: scope=cluster | Note                         |
+|-----------------------------|---------------------------|--------------------------|------------------------------|
+| #Nodes                      | n/a                       | 5000                     |                              |
+| #Namespaces                 | n/a                       | 10000                    |                              |
+| #Pods                       | 3000                      | 150000                   |                              |
+| #Pods per node              | min(110, 10*#cores)       | min(110, 10*#cores)      |                              |
+| #Services                   | 5000                      | 10000                    |                              |
+| #All service endpoints      | TBD                       | TBD                      |                              |
+| #Endpoints per service      | 250                       | n/a                      |                              |
+| #Secrets                    | TBD                       | TBD                      |                              |
+| #ConfigMaps                 | TBD                       | TBD                      |                              |
+| #Deployments                | 2000                      | TBD                      |                              |
+| #DaemonSets                 | TBD                       | TBD                      |                              |
+| #Jobs                       | TBD                       | TBD                      |                              |
+| #StatefulSets               | TBD                       | TBD                      |                              |
+| #AccessTokens               | 2000                      | 2000                     |                              |
+| #AccessTokens verifications | 5000                      | 5000                     | QPS, not a number of objects |
```

Comment:
Let's not add additional column just for this comment.

How about just making "5000 QPS" instead?

Reply:
SG. Removed column and changed '5000' -> '5000 QPS'.